### PR TITLE
Release notes.

### PR DESCRIPTION
**Pull request by dlorenc**
Ref #162

### CODE REVIEW

**Comment by dlorenc** [comment]
cc @luxas @aaron-prindle 
PTAL and let me know what's missing.

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Minikube Release Notes
+
+## Version 0.2.0 - 6/3/2016
+ * conntrack is now bundled in the ISO.
+ * DNS is now fixed.
```
Comment (luxas) [inline RELEASE-NOTES.md]: Maybe "is now working?"

**Comment by luxas** [comment]
I would prefer CHANGELOG.md instead, in order to be consistent with main kubernetes

**Comment by dlorenc** [comment]
Changed the name and fixed the wording. Also added the minikube ssh command.

**Comment by luxas** [comment]
Add
 - Dramatically improved integration testing
 - Switched to glog logging instead of fmt.Printf
 - Updated Kubernetes to _commitsha_ if we decide to merge it

**Comment by dlorenc** [comment]
Added.

**Comment by aaron-prindle** [comment]
LGTM

**Pull request merged**